sdk/core/Azure.Core/tests/ArrayBackedPropertyBagTests.cs

```diff
@@ -167,14 +167,8 @@ public void Dispose(int count)
             }
 
             target.Dispose();
-#if DEBUG
-            Assert.Throws<ObjectDisposedException>(() => { _ = target.Count; });
-            Assert.Throws<ObjectDisposedException>(() => { _ = target.IsEmpty; });
-            for (var key = 0; key < count; key++)
-            {
-                Assert.Throws<ObjectDisposedException>(() => { _ = target.TryGetValue(key, out _); });
-            }
-#else
+
+#if !DEBUG
```

Comment:
Is this related to the version bump?

Reply:
Yes. Test fails in combination "Azure.Core - Release" + "Azure.Core.Test - Debug" which becomes possible when Azure.Core is referenced as package